buildSrc/src/main/kotlin/org/sonarsource/kotlin/buildsrc/tasks/CreateRuleStubsTask.kt

```diff
@@ -35,6 +31,10 @@ abstract class CreateRuleStubsTask : DefaultTask() {
     @get:Optional
     val message: String? by project
 
+    @get:Input
+    @get:Optional
+    val gradleRule: String? by project
```

Comment:
Replace with separate task as explained in file-level comment above. Don't hesitate to reach out if you have questions about how to do this.